src/client.h

```diff
@@ -368,10 +368,15 @@ class CClient : public QObject
     int                     iServerSockBufNumFrames;
 
     // for ping measurement
-    CPreciseTime            PreciseTime;
+    QElapsedTimer           PreciseTime;
```

Comment:
Why did you change this? Was the ping time incorrect? How does this change improve the functionality?

Reply:
I saw QElapsedTimer used another place and thought there is no need to duplicate the OS checks for various timing implementations. No functional difference.

Reply:
I checked: src/corelib/kernel/qelapsedtimer_mac.cpp, src/corelib/kernel/qelapsedtimer_win.cpp

Mac implementation is the same like yours (nanoseconds) Windows is no worse either (milliseconds).